ts/features/itwallet/playgrounds/components/ItwTrialSystemSection.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  ButtonSolid,
+  ListItemHeader,
+  ListItemInfo,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { constNull } from "fp-ts/lib/function";
+import * as React from "react";
+import { SubscriptionStateEnum } from "../../../../../definitions/trial_system/SubscriptionState";
+import I18n from "../../../../i18n";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import {
+  trialSystemActivationStatus,
+  trialSystemActivationStatusReset,
+  trialSystemActivationStatusUpsert
+} from "../../../trialSystem/store/actions";
+import {
+  isLoadingTrialStatusSelector,
+  isUpdatingTrialStatusSelector,
+  trialStatusSelector
+} from "../../../trialSystem/store/reducers";
+import { ITW_TRIAL_ID } from "../../common/utils/itwTrialUtils";
+
+export const ItwTrialSystemSection = () => {
+  const dispatch = useIODispatch();
+  const status = useIOSelector(trialStatusSelector(ITW_TRIAL_ID));
+  const isLoading = useIOSelector(isLoadingTrialStatusSelector(ITW_TRIAL_ID));
+  const isUpdating = useIOSelector(isUpdatingTrialStatusSelector(ITW_TRIAL_ID));
+
+  const subscribe = () => {
+    dispatch(trialSystemActivationStatusUpsert.request(ITW_TRIAL_ID));
+  };
+
+  const refresh = () => {
+    dispatch(trialSystemActivationStatus.request(ITW_TRIAL_ID));
+  };
+
+  const reset = () => {
+    dispatch(trialSystemActivationStatusReset(ITW_TRIAL_ID));
+  };
+
+  return (
+    <>
+      <ListItemHeader label="Trial" />
+      <ListItemInfo label="Current status" value={status ?? "Not present"} />
```

Comment:
What do you think to use locale instead plain strings?

Reply:
Does it make sense to use i18n in playgrounds? I mean, since it is only used by developers, it should use English. What do you think? This also helps reduce the local file size